test/code/providers/Container_ImageInventory_Class_Provider_UnitTest.cpp

```diff
@@ -155,14 +155,15 @@ class ImageInventoryTest : public CppUnit::TestFixture
 		// Should have same number of images
 		CPPUNIT_ASSERT_EQUAL(images.size(), context.Size());
 
-		wchar_t currentId[66];
+		wchar_t currentId[72];
```

Comment:
I hate buffers like this (just big enough). How about something PLENTY big enough, like 128 or something?